services/movie_session.py

```diff
@@ -42,3 +42,8 @@
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
```

Comment:
The type hint `list[dict]` is only supported in Python 3.9 and later. Ensure your environment is using a compatible Python version.